packages/core/src/module.js

```diff
@@ -45,11 +45,9 @@ export default class ModuleContainer {
       throw new Error('Template src not found: ' + src)
     }
 
-    // Generate unique and human readable dst filename
-    const dst =
-      template.fileName ||
-      path.basename(srcPath.dir) + `.${srcPath.name}.${hash(src)}` + srcPath.ext
-
+    const fileName = template.fileName || template.filename
```

Comment:
Neat: Adding an extra line before next comment makes it more readable. Also is it for backward support of both casings or simply DX? Maybe could be good idea to also document it if it is a new feture. A module may use `filename` which is unsupported for < 2.9

Reply:
It's mostly for DX since I see myself always writing `filename` instead of `fileName`.

What about documenting `filename` only for Nuxt `v3.X`?

Reply:
Hmm more DX taste it is. Other places we always use camelCase :) Maybe we can just add a comment above this that we _do_ support both

Reply:
Comment added :)